### PR TITLE
9.0 porting stock_picking_invoice_link

### PR DESCRIPTION
**Pull request by eLBati**
As the invoincing flow has chenged in odoo version 9,
invoices linked to pickings (and vice versa), are shown independently from
the invoicing method

### CODE REVIEW

File: stock_picking_invoice_link/README.rst [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
 
-This module adds a link between pickings and generated invoices.
-So that user can easily reach the invoice related to the picking
-and see the pickings related to the invoice. This only applies
-on the invoices generated from pickings.
+This module adds a link between pickings and invoices.
+So that user can easily reach the invoices related to the picking
+and see the pickings related to the invoice.
+
+As the invoincing flow has chenged in odoo version 9,
```
Comment (rvalyi) [inline stock_picking_invoice_link/README.rst]: chenged/changed

**Comment by pedrobaeza** [comment]
Please check latest commits of this module in 8.0

**Comment by eLBati** [comment]
@pedrobaeza I didn't retrieve commits because: https://github.com/OCA/stock-logistics-workflow/pull/246/commits/12e10568fbb1b8de4429a3bc1e57e6d4c954576d
But changes should be all here.
Do you see anything missing?

**Comment by eLBati** [comment]
@rvalyi thanks, done

**Comment by pedrobaeza** [comment]
Yeah, but you should resolve that conflicts, because you're overwriting commit history, attributing you code that was not made by you.

**Comment by eLBati** [comment]
We could ask the authors of the overwritten commits if they are interested in making the cherry-pick work

**Comment by lasley** [comment]
@eLBati - If you cherry pick, they will still be attributed. Only the commits you make are attributed to you 😉

**Comment by eLBati** [comment]
@lasley I know.
I followed the procedure described at https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0#if-the-module-already-exists-in-the-90-branch-but-it-doesnt-contain-all-the-commit-history-or-youre-not-sure-if-it-have-it
and got https://github.com/OCA/stock-logistics-workflow/pull/246/commits/12e10568fbb1b8de4429a3bc1e57e6d4c954576d

For me, it is not worth spending time to make the cherry-pick work.
This is why I would ask the authors of the overwritten commits if they are interested in making the cherry-pick work

**Comment by pedrobaeza** [comment]
This is not a question of worth or not worth, but a respect to past contributors. This is like resolving a conflict rebasing a branch. You need to handle the conflict, and continue cherry-picking. It's something that it shouldn't take too much to do it, but if you don't want to do it, then I'll close this branch and wait for someone (or me) to do it.

I personally think that it's worst to handle the conflicts after with your own installed fork when the module finally gets to OCA, but it's your decision.

**Comment by eLBati** [comment]
New PR , with commit history, at https://github.com/OCA/stock-logistics-workflow/pull/261
Thanks to @Hendrixcosta

**Pull request closed without merging**